diff --git a/node_modules/bitcore-lib/lib/transaction/transaction.js b/node_modules/bitcore-lib/lib/transaction/transaction.js
index 3d61478..d131e06 100644
--- a/node_modules/bitcore-lib/lib/transaction/transaction.js
+++ b/node_modules/bitcore-lib/lib/transaction/transaction.js
@@ -210,8 +210,6 @@ Transaction.prototype.uncheckedSerialize = Transaction.prototype.toString = func
 Transaction.prototype.checkedSerialize = function(opts) {
   var serializationError = this.getSerializationError(opts);
   if (serializationError) {
-    serializationError.message += ' - For more information please see: ' +
-      'https://github.com/bitpay/bitcore/blob/master/packages/bitcore-lib/docs/transaction.md#serialization-checks';
     throw serializationError;
   }
   return this.uncheckedSerialize();
