diff --git a/node_modules/bitbox-sdk/lib/TransactionBuilder.js b/node_modules/bitbox-sdk/lib/TransactionBuilder.js
index 4590229..c287ac7 100644
--- a/node_modules/bitbox-sdk/lib/TransactionBuilder.js
+++ b/node_modules/bitbox-sdk/lib/TransactionBuilder.js
@@ -79,6 +79,14 @@ var TransactionBuilder = /** @class */ (function () {
         var witnessScript;
         this.transaction.sign(vin, keyPair, redeemScript, hashType, value, witnessScript, signatureAlgorithm);
     };
+    TransactionBuilder.fromTransaction = function(transaction, network) {
+        if (network === "bitcoincash" || network === "mainnet")
+            bitcoincash = coininfo.bitcoincash.main;
+        else
+            bitcoincash = coininfo.bitcoincash.test;
+        var bitcoincashBitcoinJSLib = bitcoincash.toBitcoinJS();
+        return Bitcoin.TransactionBuilder.fromTransaction(transaction, bitcoincashBitcoinJSLib);
+    };
     TransactionBuilder.prototype.build = function () {
         if (this.p2shInput === true)
             return this.tx;
diff --git a/node_modules/bitbox-sdk/lib/TransactionBuilder.ts b/node_modules/bitbox-sdk/lib/TransactionBuilder.ts
index b3b4b15..c25d208 100644
--- a/node_modules/bitbox-sdk/lib/TransactionBuilder.ts
+++ b/node_modules/bitbox-sdk/lib/TransactionBuilder.ts
@@ -108,6 +108,16 @@ export class TransactionBuilder {
     this.transaction.setLockTime(locktime)
   }
 
+  public static fromTransaction(transaction: any, network: string): TransactionBuilder {
+    let bitcoincash;
+    if (network === "bitcoincash" || network === "mainnet")
+      bitcoincash = coininfo.bitcoincash.main
+    else bitcoincash = coininfo.bitcoincash.test
+
+    const bitcoincashBitcoinJSLib: any = bitcoincash.toBitcoinJS()
+    return Bitcoin.TransactionBuilder.fromTransaction(transaction, bitcoincashBitcoinJSLib);
+  }
+  
   public sign(
     vin: number,
     keyPair: ECPair,
